extensions/interactions/ItemSelectionInput/ItemSelectionInput.js

```diff
@@ -97,3 +98,23 @@ oppia.directive('oppiaShortResponseItemSelectionInput', [
     };
   }
 ]);
+
+oppia.factory('itemSelectionInputRulesService', [function() {
+  return {
+    Equals: function(answer, inputs) {
+      return answer.length == inputs.x.length && answer.every(function(val) {
```

Comment:
Technically, you may want to check that each set is a (not necessarily proper) subset of the other -- either that, or normalize first to remove duplicates.